docs/ref/modules/vulnerability-scanner/events.md

```diff
@@ -0,0 +1,670 @@
+# Events format 
+
+The incoming events from the agents must be parseable by the [flatbufer schemas](flatbuffers.md), otherwise it will trigger an exception and the event won't reach the vulnerability scanner. Below we detail the format for the different systems supported by the scanner.
+
+## Syscollector event types
+
+The vulnerability scanner as an event-driven module, reacts to the syscollector events sent by the agents. There are two types of events received by the scanner: synchronization and delta events. 
+
+Synchronization events are sent during the first scan of the Syscollector scan, or if a delta event fails. 
+Delta events are the ones sent from the second Syscollector scan. 
+
+those event types provide the same information to the scanner, the detection is not affected by the type of the event sent, they are just parsed differently.
+
+Within synchronization events, an **integrity_clear** event will remove the information for certain provider. For the case of package provider it will remove all vulnerabilities related to packages, for OS will remove vulnerabilities related to OS but also affect the detection for packages, since the source operating system is a required information for detection.
```

Comment:
```suggestion
Within synchronization events, an **integrity_clear** event will remove the information for a certain provider. For the case of a package provider, it will remove all vulnerabilities related to packages, for OS will remove vulnerabilities related to OS, but also affect the detection for packages, since the source operating system is a required information for detection.
```